models.rb

```diff
@@ -136,3 +136,19 @@ def validate
     errors.add(:name, 'cannot be empty') if !name || name.empty?
   end
 end
+
+# SKU stats data model
+# primary_key :id
+# String      :family
+# String      :sku_num
+# String      :description
+# Float       :rate
+# Boolean     :active, default: true
+class Sku < Sequel::Model
+  many_to_many :node_resources_projects
```

Comment:
These should be a one_to_one relationship.
Also dont forget to add the one_to_one relationships inside the NodeResourcesProjects and DbResourceProjects class